proxy-node-acceptance-tests/features/step_definitions/user_steps.rb

```diff
@@ -17,7 +17,8 @@
 end
 
 And('they progress through verify') do
-  find('label', :text => 'I’ve used GOV.UK Verify before').click
+  assert_text('Sign in with GOV.UK Verify')
+  find('label', :text => 'I\'ve already signed up for Verify').click
```

Comment:
Oh no, we should fix the quote mark to be curly :-o

Reply:
updated the test to assert an id element